Controller/WebsiteCommentController.php

```diff
@@ -118,17 +118,31 @@ public function cgetCommentsAction(string $threadId, Request $request): Response
     {
         list($type, $entityId) = $this->getThreadIdParts($threadId);
 
-        $page = $request->get('page');
+        $limit = $request->query->getInt('limit') ?? 10;
+        $offset = $request->query->getInt('offset') ?? 0;
+
+        $pageSize = $request->get('pageSize') ?? 10;
+        if ($pageSize) {
+            @\trigger_deprecation('sulu/comment-bundle', '2.x', 'The usage of the "pageSize" parameter is deprecated.
+        Please use "limit" and "offset instead.');
+            $limit = $pageSize;
+        }
+
+        $page = $request->get('page') ?? null;
+        if ($page) {
```

Comment:
`?? null` is unnecessary `??` says `if value == null` and if it is already null we don't need to set it to null.

```suggestion
        $page = $request->get('page');
        if ($page) {
```